EduVPN/Helpers/UserDefaults+Preferences.swift

```diff
@@ -57,6 +58,18 @@ extension UserDefaults {
         }
     }
 
+    var isStatusItemInColor: Bool {
+        get {
+            if object(forKey: Self.isStatusItemInColorKey) == nil {
+                return true // Default to true
+            }
```

Comment:
You don't need this. You can register defaults with NSUserDefaults.

Reply:
Actually, we want the default to be grayscale, so we don't need that `if` block at all. Removed it in 
commit f214205.

That said, we do use this pattern for showInStatusBar and showInDock. The reason why `UserDefaults.register(defaults:)` isn't used there is that there's no good place to call it from. We could do it from the AppDelegate, but that's not a great place for this. So I used this pattern, so that the default values are present close to the code that reads the values. What do you think?

Reply:
I normally do that in the app delegate indeed. Not sure why that's not a good place. You could move it to a method in this helper and call that from the app delegate.

Reply:
> Not sure why that's not a good place.

I felt it contributes to applicationDidFinishLaunching having a bunch of random code. But I don't have a strong opinion on it, so I'll modify it as you suggested.

Reply:
Resolved in commit f214205.